### ISSUE TITLE
Can I use mapbox navigation to create an indoor navigation?

### ISSUE THREAD

**Issue by phfeitosaa** [mapbox/mapbox-navigation-android#31]
How could i use sdk navigation to create a navigation inside an indoor map? Let me explain better, I need to create a navigation application for internal locations that shows the route and navigates from one place to another. To make the location of the user without using the gps, I will use qr codes scattered throughout the building to give me the exact location of the user and then show the route and make the navigation to the destination. But I'm not sure if it's possible to do this using the mapbox, since it's geared towards external applications.

I need to create something like this:
![image](https://cloud.githubusercontent.com/assets/12596115/25781940/f6db2332-3316-11e7-8059-ce6a255bbf78.png)

I would be grateful if anyone could help me.

**Comment by cammace** [comment]
Hey @phfeitosaa, while indoor navigation would be awesome to implement, several issues would need to be resolved and I'm not sure how it would work with QR codes instead of GPS. Since we are in the early days with this project, its safe to say that in the near future, we have no plans to support indoor navigation.

**Comment by Aft3rmathpwnz** [comment]
Hi there, I'm building indoor navigation app, before I was using SVG files and rendered primitives by myself, now I'm trying to find some complete solution for indoor map rendering, so not to open new issue: can we determine user's coordinates by ourselves (BLE+sensors) and then just assign it to the current location pin's coordinates in the scale of map? Thanks.

**Comment by indurtividya** [comment]
@Aft3rmathpwnz I think Mapwize is a better solution in terms of indoor navigation. Mapbox is more for routing outdoors using their existing routes data.

**Comment by JERINJACOBCHERIAN** [comment]
Hi,Do you have a solution now? I was trying to implement it,but cant figure out how to assign geolocation to indoor floor plans

**Comment by Aft3rmathpwnz** [comment]
Hi, unfortunately no, we decided to use SVG and was parsing it by our own.

**Comment by JERINJACOBCHERIAN** [comment]
oh. so what about this tutorial https://blog.mapbox.com/indoor-navigation-in-ar-with-unity-6078afe9d958 ?

**Comment by DivyaVaniChinta** [comment]
> oh. so what about this tutorial https://blog.mapbox.com/indoor-navigation-in-ar-with-unity-6078afe9d958 ?

@JERINJACOBCHERIAN  , the above link is for indoor navigation for different platform (UNITY). I tried to use this in **Xamarin.Forms**, But this sample code project structure is different 
![image](https://user-images.githubusercontent.com/29367392/89606605-ff186a80-d88d-11ea-86cd-34b2e126dcde.png)

I checked MapBox Docs, They have given one example , native code for styling indoor map into 3D. Thats it. Apart from that I could not find any info about indoor navigation in detail implementation.

  **Can anyone confirm that whether MapBox supports Indoor Navigation implementation in Xamarin.Forms ?**
